contrib/tinyxml2/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "tinyxml2"
+pkgver = "10.0.0"
+pkgrel = 0
+build_style = "cmake"
+configure_args = ["-Dtinyxml2_SHARED_LIBS=1"]
```

Comment:
```suggestion
configure_args = ["-DBUILD_SHARED_LIBS=ON"]
```

since it reads the standard arg. also we use on/off for all of these for cmake as the bool value